test/objectwrap_constructor_exception.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+const buildType = process.config.target_defaults.default_configuration;
+const assert = require('assert');
+
+const test = (binding) => {
+  const { ConstructorExceptionTest } = binding.objectwrapConstructorException;
+  assert.throws(() => (new ConstructorExceptionTest()));
```

Comment:
Could this be improved to validate that we get the same exception as we expect?

Reply:
Can do.